### PR TITLE
Adds fireproof masks, and tweaks flameproof atmos clothing

### PR DESCRIPTION
**Pull request by carlarctg**
## About The Pull Request
Adds the atmospheric gas mask, which is flameproof, has very small radiation protection, and shows up in atmospheric lockers and suit storage units.
image
Captain gets a similar mask, but it's acidproof too.
image
Standarized captain's jetpack name

## Why It's Good For The Game

This [fixes the bug where the captain's hardsuit wasn't flameproof](https://github.com/tgstation/tgstation/issues/42005) via giving the captain a fireproof mask, since before, the hardsuit not having HIDEMOUTH made it so gas went in through your mouth hole.
Atmos gas masks make sense, and it feels a tad odd for the elite firefighters to use the same gas masks every tider takes.
Cap gets a gas mask and not a hailer to subtly discourage capsec.

## Changelog wow i sure am happy it works thank you mso
:cl:
add: Atmos techs now have their unique gas masks
add: captain does too
fix: captain hardsuit finally fireproof if worn with captain gas mask
tweak: removes useless spare analyzer from atmos locker
/:cl:

### CODE REVIEW

**Comment by Detective-Google** [comment]
These look pretty sick

**Comment by kopoba** [comment]
Welding gas mask nerf =\

**Comment by Fikou** [comment]
why was caps swat suit name changed??

**Comment by carlarctg** [comment]
swat suit sounds a lot dumber than hardsuit

i accidentally lowered the melee by five haha whoops

tint i already said in the main body

**Comment by Fikou** [comment]
the melee protection is worse on the helmet too
swatsuit stays consistent with the rest of the cool suits, and you didnt change the helmet name
why make the tint higher on the worse item, the firefighting helmet instead of the better, atmos hardsuit which is spaceproof???
why even nerf firefighting helmet and suit giving the helmet welding sight makes it worse than a regular firefighting helmet

File: code/modules/clothing/spacesuits/hardsuit.dm [hunk 1]
```diff
@@ -186,12 +186,14 @@
 	//Atmospherics
 /obj/item/clothing/head/helmet/space/hardsuit/engine/atmos
 	name = "atmospherics hardsuit helmet"
-	desc = "A special helmet designed for work in a hazardous, low-pressure environment. Has thermal shielding."
+	desc = "A special helmet designed for work in a hazardous, low- or high-pressure environment. Has thermal shielding, but the same shielding makes it unsuitable for welding work."
```
Comment (ShizCalev) [inline code/modules/clothing/spacesuits/hardsuit.dm]: low- or high-pressure

???????

**Comment by necromanceranne** [comment]
Why is there a ton of undocumented changes in this PR? The welding mask has 10 less melee armor? The kinda suspect initial armor value changes on the hardsuit?

Document your changes dude, and explain why those exist in the first place.

**Review by 81Denton — CHANGES_REQUESTED** [review]
Please split the new gas masks and the balance changes into two separate PRs

**Comment by carlarctg** [comment]
bluh

**Comment by carlarctg** [comment]
> Why is there a ton of undocumented changes in this PR? The welding mask has 10 less melee armor? The kinda suspect initial armor value changes on the hardsuit?
> 
> Document your changes dude, and explain why those exist in the first place.
> 
> Also, the reason it's called a MK.II Swat Suit is because it's in between the MK.I swat suit (the one you get from cargo, the big fat suit), and the MK.III, which is Deathsquad armor. It's a suit of armor, it's not like hardsuits which are kind of supposed to be industrial rig suits fitted to serve various purposes across the station (medical for medical purposes, security for firefights, etc). The cap's suit is a standout. It's entirely and only meant to be a suit of armor that is spaceproof. It has a different intended purpose and origin, basically.

sorry for the lack of documentation
welding mask nerf because 1. it's dumb 2. i was in the gas mask dm so i thought "may as well", will remove
captain's "swat" suit got 80 less fire res on the helmet because the last 20 is added by the gas mask, totalling 100
the lore's kind of neat and i did notice it, but i just renamed the hardsuit to that because it already had the helmet named as 'captain's hardsuit helmet' so i just made it consistent, in the way i thought was better.  i'll keep the SWAT flavor in the description

File: code/modules/clothing/spacesuits/hardsuit.dm (hunk 1 above)
Comment (carlarctg) [reply code/modules/clothing/spacesuits/hardsuit.dm]: myeh

Comment (PKPenguin321) [reply code/modules/clothing/spacesuits/hardsuit.dm]: `extreme-pressure` is the phrase you're looking for

**Comment by ATH1909** [comment]
> > Why is there a ton of undocumented changes in this PR? The welding mask has 10 less melee armor? The kinda suspect initial armor value changes on the hardsuit?
> > Document your changes dude, and explain why those exist in the first place.
> > Also, the reason it's called a MK.II Swat Suit is because it's in between the MK.I swat suit (the one you get from cargo, the big fat suit), and the MK.III, which is Deathsquad armor. It's a suit of armor, it's not like hardsuits which are kind of supposed to be industrial rig suits fitted to serve various purposes across the station (medical for medical purposes, security for firefights, etc). The cap's suit is a standout. It's entirely and only meant to be a suit of armor that is spaceproof. It has a different intended purpose and origin, basically.
> 
> sorry for the lack of documentation
> welding mask nerf because 1. it's dumb 2. i was in the gas mask dm so i thought "may as well", will remove
> captain's "swat" suit got 80 less fire res on the helmet because the last 20 is added by the gas mask, totalling 100
> the lore's kind of neat and i did notice it, but i just renamed the hardsuit to that because it already had the helmet named as 'captain's hardsuit helmet' so i just made it consistent, in the way i thought was better. i'll keep the SWAT flavor in the description

Uh, you do know how fire armor for clothing works, right? IIRC, a piece of clothing's fire armor is how resistant THE PIECE OF CLOTHING ITSELF is to fire, NOT how much protection from fire gives its WEARER (correct me if I'm wrong, though).

**Comment by ATH1909** [comment]
Also, why not just make hardsuits protect masks from fire damage?

File: code/modules/clothing/spacesuits/hardsuit.dm [hunk 2]
```diff
@@ -190,7 +190,7 @@
 	icon_state = "hardsuit0-atmospherics"
 	item_state = "atmo_helm"
 	hardsuit_type = "atmospherics"
-	armor = list("melee" = 30, "bullet" = 5, "laser" = 10, "energy" = 5, "bomb" = 10, "bio" = 100, "rad" = 25, "fire" = 100, "acid" = 75)
+	armor = list("melee" = 30, "bullet" = 5, "laser" = 10, "energy" = 5, "bomb" = 10, "bio" = 100, "rad" = 50, "fire" = 100, "acid" = 75)
```
Comment (Fikou) [inline code/modules/clothing/spacesuits/hardsuit.dm]: why is this changed without changing the suit part

File: code/modules/clothing/spacesuits/hardsuit.dm [hunk 3]
```diff
@@ -537,7 +537,7 @@
 	icon_state = "swat2helm"
 	item_state = "swat2helm"
 	desc = "A tactical SWAT helmet MK.II."
-	armor = list("melee" = 40, "bullet" = 50, "laser" = 50, "energy" = 25, "bomb" = 50, "bio" = 100, "rad" = 50, "fire" = 100, "acid" = 100)
+	armor = list("melee" = 40, "bullet" = 50, "laser" = 50, "energy" = 25, "bomb" = 50, "bio" = 100, "rad" = 40, "fire" = 80, "acid" = 100)
```
Comment (Fikou) [inline code/modules/clothing/spacesuits/hardsuit.dm]: ditto

File: code/modules/clothing/spacesuits/hardsuit.dm [hunk 4]
```diff
@@ -566,11 +566,11 @@
 	name = "captain's hardsuit helmet"
 	icon_state = "capspace"
 	item_state = "capspacehelmet"
-	desc = "A tactical MK.II SWAT helmet boasting better protection and a horrible fashion sense."
+	desc = "A tactical MK.II SWAT helmet boasting better protection and a reasonable fashion sense."
 
 /obj/item/clothing/suit/space/hardsuit/swat/captain
-	name = "captain's SWAT suit"
-	desc = "A MK.II SWAT suit with streamlined joints and armor made out of superior materials, insulated against intense heat. The most advanced tactical armor available. Usually reserved for heavy hitter corporate security, this one has a regal finish in Nanotrasen company colors. Better not let the assistants get a hold of it."
+	name = "captain's hardsuit"
```
Comment (Fikou) [inline code/modules/clothing/spacesuits/hardsuit.dm]: would be better as "captain's SWAT Suit" with the helmet being changed to "captain's SWAT Helmet"

**Review by Fikou — CHANGES_REQUESTED** [review]

File: code/modules/clothing/spacesuits/hardsuit.dm (hunk 3 above)
Comment (carlarctg) [reply code/modules/clothing/spacesuits/hardsuit.dm]: i don't even remember doing the rad part, but i just said why on the fire thing

**Comment by carlarctg** [comment]
@Fikou are you HAPPY NOW

**Comment by Fikou** [comment]
> @Fikou are you HAPPY NOW

yes i liked ;-)

File: code/modules/clothing/spacesuits/hardsuit.dm (hunk 2 above)
Comment (carlarctg) [reply code/modules/clothing/spacesuits/hardsuit.dm]: fixed ages ag

File: code/game/objects/structures/crates_lockers/closets/secure/engineering.dm [hunk 5]
```diff
@@ -91,10 +91,10 @@
 	new /obj/item/pipe_dispenser(src)
 	new /obj/item/storage/toolbox/mechanical(src)
 	new /obj/item/tank/internals/emergency_oxygen/engi(src)
-	new /obj/item/analyzer(src)
```
Comment (ExcessiveUseOfCobblestone) [inline code/game/objects/structures/crates_lockers/closets/secure/engineering.dm]: is this because there's already one in the toolbox? undoc'd change

Comment (carlarctg) [reply code/game/objects/structures/crates_lockers/closets/secure/engineering.dm]: i really need to stop doing undocumented changes aaaaaaaaaaa

yes cobby, it's pretty useless so i decided to remove it @ExcessiveUseOfCobblestone

Comment (ExcessiveUseOfCobblestone) [reply code/game/objects/structures/crates_lockers/closets/secure/engineering.dm]: when I review I get notifs w/o need to ping btw

Comment (carlarctg) [reply code/game/objects/structures/crates_lockers/closets/secure/engineering.dm]: but it took you 20 hrs to respond.....................

Comment (ExcessiveUseOfCobblestone) [reply code/game/objects/structures/crates_lockers/closets/secure/engineering.dm]: eat / breathe / sleep

ss13
